api/src/test/java/org/openmrs/module/xdssender/api/notificationspullpoint/impl/NotificationsPullPointClientImplTest.java

```diff
@@ -1,33 +0,0 @@
-package org.openmrs.module.xdssender.api.notificationspullpoint.impl;
```

Comment:
@moshonk Do you think we could keep this test file? I'm not sure how time-consuming or difficult it would be, but ideally, we would have at least two tests, one for `getNewMessages(Location currentLocation)` and one for`getNewMessages()`, that capture this work.

Reply:
The suggestion sounds good @pmanko. I'll restore the test file and work on the 2 tests.